### DISCUSSION TITLE
Bedrock 1.17

### DISCUSSION THREAD

**Discussion by JRBros2346** [bridge-core/bridge.#372] [Q&A]
Will this be updated to 1.17?
If so pls let me know

**Comment by solvedDev** [comment]
For full support, please use bridge. v2: https://editor.bridge-core.app

We have support for most 1.17 features on bridge. v1 already though. Anything specifically you're looking for?

**Reply by JRBros2346** [reply]
It says it is not a stable release. Is it okay to use it for 1.17? Does it have all features? Also in minecraft wiki when 1.17 was in beta, it told that there maybe some changes in the stable release.

**Reply by solvedDev** [reply]
bridge. v2 is available as a stable release since Sunday
